.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -5,13 +5,37 @@ description: Create a report to help us improve
 body:
 - type: markdown
   attributes:
-    value: |
+    value: >
+      **Thank you for wanting to report a bug in ansible-core!**
+
+
       ⚠
-      Verify first that your issue is not [already reported on GitHub][issue search].
+      Verify first that your issue is not [already reported on
+      GitHub][issue search] and keep in mind that we may have to keep
+      the current behavior because [every change breaks someone's
+      workflow][XKCD 1172].
+      We try to be mindful about this.
```

I don't think the xkcd helps people.
```suggestion
      Verify first that your issue is not [already reported on
      GitHub][issue search].
```

---

Yeah, my intention was to try making the texts more humane to give a feeling that the receiving end is actually a bunch of humans. But maybe I misjudged where the line between welcoming and sarcastic lays...

---

It's a really difficult thing to judge. I'm not sure I get it right, which is why I generally personally try and avoid humor.

---

I think that the general public is well familiar with this comic. And I also hoped to urge people not to be blind to the consequences/price of fixing bugs.

Any ideas on how we could preserve such a hint?

---

@gundalow some folks on slack said that it'd be nice to have a little bit of humor. I think it doesn't hurt either. How strongly opposed are you to this reference? Is there anything we can do to make sure that the intent for this not to sound insulting in any way is explicit and clear?

---

Keep it then.